### PR TITLE
Update index.md Domain name alias

### PR DESCRIPTION
**Pull request by emirgens**

### CODE REVIEW

File: public-site/docs/src/docs/topic-domain-names/index.md [hunk 1]
```diff
@@ -70,6 +70,22 @@ Examples:
...
 - `oneapp.app.radix.equinor.com`
 
+## App alias
+
+```text
+[appName].[clusterEnvNamepace].radix.equinor.com
+```
+`dnsAlias` creates one or several DNS aliases in the form of `<alias>.radix.equinor.com` for the specified environment and component. There are few reserved aliases which cannot be used:
+
+- Only allocated for **active clusters**
```
Comment (emirgens) [inline public-site/docs/src/docs/topic-domain-names/index.md:80]: Remove this, not applicable for users

**Review by emirgens — COMMENTED** [review]
approved

**Review by satr — APPROVED** [review]

**Review by satr — APPROVED** [review]

**Review by Richard87 — APPROVED** [review]
godkjent

File: public-site/docs/src/docs/topic-domain-names/index.md (hunk 1 above)
Comment (satr) [reply public-site/docs/src/docs/topic-domain-names/index.md]: Fixed

**Pull request merged**